fetch_rapids.cmake

```diff
@@ -11,6 +11,13 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
+include(FetchContent)
+FetchContent_Declare(
+  rapids-cmake
+  GIT_REPOSITORY https://github.com/robertmaynard/rapids-cmake.git
```

Comment:
Is this a temporary repo path?

Reply:
Yes, waiting on https://github.com/rapidsai/rapids-cmake/pull/190 to be merged.

Reply:
rapids-cmake PR has been merged, so this has been removed.